style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+  overflow: hidden;
+}
+
+:root {
+  --base: #ffc600;
+  --spacing: 10px;
+  --blur: 0px;
+  --grayscale: 0%;
+  --invert: 0%;
+  --saturate: 100%;
+  --hue-rotate: 0;
+  --sepia: 0%;
+}
+
+img {
+  padding: var(--spacing);
+  background: var(--base);
+  filter: blur(var(--blur)) grayscale(var(--grayscale)) invert(var(--invert)) saturate(var(--saturate)) hue-rotate(var(--hue-rotate)) sepia(var(--sepia));
+  margin: 0 5%;
+  width: 100%;
+}
+
+.hl {
+  color: var(--base);
+}
+
+h2 {
+  text-align: center;
+  margin-bottom: 5%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+  background: #193549;
+  color: white;
+  font-family: "helvetica neue", sans-serif;
+  font-weight: 100;
+  font-size: 20px;
+}
+
+.container {
+  margin: 0 100px;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+}
+
+.controls {
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  width: 90%;
+}
+
+@media (max-width: 767px) {
+  .container {
+    flex-direction: column;
+    margin: 0;
+  }
+  .controls {
+    padding-right: 36%;
+    height: 195px;
+    flex-wrap: wrap
+  }
+}
```

Comment:
There is a problem with the whole layout: 
on my screen the image doesn't fit the screen height, the image is getting cut from bottom. For the layout like this try to implement the markup so that the whole content fit the screen.


![image](https://user-images.githubusercontent.com/45519864/119225977-9b56e980-bb0f-11eb-8674-dd1d189569b2.png)

Reply:
DONE!